integrations/sonarqube/client.py

```diff
@@ -97,6 +108,10 @@ async def send_paginated_api_request(
         json_data: Optional[dict[str, Any]] = None,
     ) -> list[dict[str, Any]]:
         try:
+            logger.info(
```

Comment:
This is great for debugging but I think it will probably spam the logs, maybe try to use debug

```suggestion
            logger.debug(
```